bites.py

```diff
@@ -1,56 +1,84 @@
 import random
 from constants import ANTS
+from constants import FOOD_TYPES
 from constants import K_COLOUR_V_FOOD_DICT
 # from constants import K_FOOD_V_COLOUR_DICT
-  
-def initialise_ants(ants):
-  """Create a record of the starting positions of each insect meeple
-  This will show the starting positions as None, since the ants are not 
-  positioned on the trail immediately.
-
-  Parameters
-  ----------
-  ants : (list)
-    A list containing the ID of each ant, refered to by colour
-    The list can be found at constants.py/ANTS
-    Each element is a string
 
-  Returns
-  -------
-  ant_positions : (dict)
-    A dictionary showing the starting location of each ant.
-    Initially set as None, the position will be changed throughout the game as 
-    the ant moves along the trail, the position on the trail will be defined as the 
-    index of the element in the trail list, from initialise_trail
-    Keys are strings
-    Values are integers
-  """
-  ant_positions = {}
-  for ant in ants:
-    ant_positions[ant] = None
-  return ant_positions
-  
-def initialise_trail(foods):
-  """Create the path of tokens that the game is played on
+class Bites():
+  def __init__(self, ants, tokens_for_trail):
```

Comment:
The Player class has documentation for the init method. Consider adding some here too.